packages/core/apps/core-apps-browser-internal/src/status/lib/load_status.ts

```diff
@@ -78,6 +73,17 @@ function formatMetrics({ metrics }: StatusResponse): Metric[] {
       value: (metrics.requests.total * 1000) / metrics.collection_interval_in_millis,
       type: 'float',
     },
+    {
+      name: i18n.translate('core.statusPage.metricsTiles.columns.utilizationHeader', {
+        defaultMessage: 'Utilization (active: {active} / idle: {idle})',
+        values: {
+          active: numeral(metrics.process.event_loop_utilization.active).format('0.00'),
+          idle: numeral(metrics.process.event_loop_utilization.idle).format('0.00'),
+        },
+      }),
+      value: metrics.process.event_loop_utilization.utilization,
+      type: 'float',
```

Comment:
We're autoformatting floats to two decimal places... it's nice to see the utilization values with a bit more precision, but I don't know if it's worth a one-off implementation just for this metric. I tried updating all floats to five decimal places, but it blew up the layout, so I decided to leave it alone since this is a nice-to-have item anyway 🤷